doc/xanadu_theme/layout.html

```diff
@@ -0,0 +1,311 @@
+    {%- endif %}
+  {%- else %}
+    {{ homepage() }}
+  {%- endif %}
+{%- endmacro %}
+
+{# Displays the URL for the projectpage if it's set or falls back to homepage macro #}
+{% macro projectlink() -%}
+  {%- if theme_projectlink %}
+    {%- if hasdoc(theme_projectlink) %}
+      {{ pathto(theme_projectlink) }}
+    {%- else %}
+      {{ theme_projectlink }}
+    {%- endif %}
+  {%- else %}
+    {{ homepage() }}
+  {%- endif %}
+{%- endmacro %}
+
+{# Displays the next and previous links both before and after content #}
+{% macro render_relations() -%}
+  {% if prev or next %}
+  <div class="footer-relations">
+    {% if prev %}
+      <div class="pull-left">
+        <a class="btn btn-xanadu" href="{{ prev.link|e }}" title="{{ _('previous chapter')}} (use the left arrow)"><i class="fas fa-arrow-left"></i> Previous</a>
+      </div>
+    {% endif %}
+    {%- if next and next.title != '&lt;no title&gt;' %}
+      <div class="pull-right">
+        <a class="btn btn-xanadu" href="{{ next.link|e }}" title="{{ _('next chapter')}} (use the right arrow)">Next <i class="fas fa-arrow-right"></i></a>
+      </div>
+    {%- endif %}
+    </div>
+    <div class="clearer"></div>
+  {% endif %}
+{%- endmacro %}
+
+{%- macro guzzle_sidebar() %}
+  <div id="left-column" class="nano css-transitions-only-after-page-load">
+    <div class="sphinxsidebar nano-content">
+      {%- if sidebars != None %}
+        {#- new style sidebar: explicitly include/exclude templates #}
+        {%- for sidebartemplate in sidebars %}
+        {%- include sidebartemplate %}
+        {%- endfor %}
+      {% else %}
+        {% include "searchbox.html" %}
+        {% include "globaltoc.html" %}
+      {%- endif %}
+    </div>
+  </div>
+{%- endmacro %}
+
+{%- block header -%}
+  {% include "header.html" %}
+{%- endblock %}
+
+{%- block content %}
+
+  {%- if pagename == 'index' and theme_index_template %}
+    {% include theme_index_template %}
+  {%- else %}
+    <div class="container-wrapper">
+      {%- block sidebar1 %}{{ guzzle_sidebar() }}{% endblock %}
+
+      {%- block document_wrapper %}
+        {%- block document %}
+        <div id="content">
+          <div id="right-column">
+            {% block breadcrumbs %}
+            <div role="navigation" aria-label="breadcrumbs navigation">
+              <ol class="breadcrumb">
+                <li><a href="{{ pathto(master_doc) }}">Docs</a></li>
+                {% for doc in parents %}
+                  <li><a href="{{ doc.link|e }}">{{ doc.title }}</a></li>
+                {% endfor %}
+                <li>{{ title }}</li>
+                {%- if show_source and has_source and sourcename %}
+                  {%- if show_on_github_url %}
+                    <li style="float: right;" class="showsource">
+                      <a href="{{ show_on_github_url }}" class="reference-external" rel="nofollow">
+                        <i class="fab fa-github"></i>
+                        {{ _('Show on GitHub') }}
```

Comment:
This is something broader than just for this plugin, but would we like to keep this? (also curious on the input from @josh146 )

Some content will be built runtime and therefore the `Show on GitHub` button will be broken. Specifically here, those that are working (I think a handful only), redirect one to `rst` files which mostly include references from other `rst` files. Also, for the sources in the API, there is a `source` option which has a working redirect (`Show on Github` breaks there as well).

Overall, I'd be in for removing it for now, but I can be convinced otherwise. :slightly_smiling_face:

Reply:
You are so right, I am also in favour of removing. Your comment made me also aware that we should redirect the Github button next to FAQ and Support in the navbar, to link to the plugin...it is confusing if you are in the plugin docs and that button links to pennylane, no?

On the downside, this would make the maintenance of the html layout more difficult (we wanted to centralise it).

Reply:
For now I changed the target of the navbar Github symbol to link to the plugin.

Reply:
I'm happy with removing the show on github button :+1:

I tend to use it on handwritten `rst` pages (e.g., the overview pages --- sometimes I click on 'Show on GitHub', make the modification there, and GitHub automates making a PR out of it). But I think I'm the only one who uses it, and I agree it makes no sense on the API pages.